en/editor/console/index.md

```diff
@@ -0,0 +1,28 @@
+# Console
+
+![main](index/main.jpg)
+
+**Console** display errors, warnings, or other log information generated by the editor and engine. Information of different important levels is displayed in different colors.
```

Comment:
The **Console** displays

Information of different importance levels is displayed
or
Information of different levels of importance are displayed